src/connect4/core.rs

```diff
@@ -44,8 +46,8 @@ const BOARD_POS_OFFSET: (i32, i32) = (10, 10 + COLUMN_SELECTION_INDICATOR_POS_OF
 
 /// Constant definition for the screen size of the game window
 const SCREEN_SIZE: (f32, f32) = (
-    BOARD_TOTAL_SIZE.0 + 32 as f32,
-    BOARD_TOTAL_SIZE.1 + (BOARD_POS_OFFSET.1 - 10 + 32) as f32,
+    BOARD_TOTAL_SIZE.0 + (BOARD_POS_OFFSET.0 as f32),
+    BOARD_TOTAL_SIZE.1 + (BOARD_POS_OFFSET.1 as f32),
```

Comment:
Resized board here so grid not cutoff...think you just made a change and forgot to add this adjustment